CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to Poutyne
+We love your input! We want to make contributing to this project as easy and transparent as possible, whether it's:
+
+- Reporting a bug
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase. We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
```

Comment:
Should we propose a branch name pattern ?
Something like: `fixing_bug-... `

Reply:
Since we suggest to fork the repository, the contributor never creates a branch on the Poutyne git, only on his personal workspace. When submitting the PR, he then proposes changes coming from his own fork and therefore branch naming is not really an issue here.

Correct me if i'm wrong @freud14 .